api/v1/openapi.yaml

```diff
@@ -0,0 +1,953 @@
+    get:
+      summary: Retrieve health of Cilium daemon
+      description: |
+        Returns the health and status of the Cilium daemon and all related
+        components.
+      tags:
+      - daemon
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/StatusResponse"
+  "/config":
+    get:
+      summary: Retrieve daemon configuration
+      description: |
+        Returns the configuration of the daemon which includes addressing
+        and configuration settings.
+      tags:
+      - daemon
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/DaemonConfigurationResponse"
+    put:
+      summary: Update daemon configuration
+      description: |
+        Updates the daemon configuration with the provided ConfigurationMap
+        and regenerates & recompiles all required datapath components.
+      tags:
+      - daemon
+      parameters:
+      - name: configuration
+        in: body
+        required: true
+        schema:
+          "$ref": "#/definitions/ConfigurationMap"
+      responses:
+        '200':
+          description: Success
+        '400':
+          description: Bad configuration parameters
+          schema:
+            "$ref": "#/definitions/Error"
+        '500':
+          description: Recompilation failed
+          x-go-name: Failure
+          schema:
+            "$ref": "#/definitions/Error"
+  "/endpoint/{id}":
+    get:
+      summary: Retrieve an endpoint
+      description: |
+        Looks up and returns the endpoint described with the endpoint ID.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/Endpoint"
+        '400':
+          description: Invalid endpoint ID format for specified type
+          x-go-name: Invalid
+          schema:
+            "$ref": "#/definitions/Error"
+        '404':
+          description: Endpoint not found
+    delete:
+      summary: Delete endpoint
+      description: |
+        Deletes the endpoint specified by the ID. Deletion is imminent and
+        atomic, if the deletion request is valid and the endpoint exists,
+        deletion will occur even if errors are encounted in the process. If
+        errors have been encountered, the code 202 will be returned, otherwise
+        200 on success.
+
+        All resources associated with the endpoint will be freed and the
+        workload represented by the endpoint will be disconnected.It will no
+        longer be able to initiate or receive communications of any sort.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      responses:
+        '200':
+          description: Success
+        '206':
+          description: Deleted with a number of errors encountered
```

Comment:
Can always be added but I want to first try and get rid of the error conditions so this is a workaround for an inconsistent API execution for now.